docs/test_runners.md

```diff
@@ -0,0 +1,34 @@
+# Elección de `test runner`
+> Objetivo 4
+
+## Criterios
+Facilidad de instalación, velocidad.
+
+## Opciones
+> Tras leer en varios foros y teniendo en cuenta las características del modelo, las siguientes son las opciones que tendremos en consideración pese a que haya muchas alternativas más.
+
+### Mocha
+* Hay muchas dudas resueltas en foros y extensa documentación.
```

Comment:
¿Esto se refiere a "facilidad de instalación" o a "velocidad"? Si no se refiere a ninguno, ¿por qué lo ponéis?

Reply:
:arrow_up: